server/resources/resource_management_test.go

```diff
@@ -2202,6 +2238,72 @@ var _ = Describe("Resource manager", func() {
 				})
 			})
 		})
+
+		addAuthAndEmptyResponseToContext := func(auth schema.Authorization) {
+			context["tenant_id"] = auth.TenantID()
+			context["domain_id"] = auth.DomainID()
+			context["auth"] = auth
+			context["response"] = ""
+		}
+
+		Context("Custom action isolation tests", func() {
+			var NoProblem resources.ResourceProblem = -1
+
+			DescribeTable("Performing action", func(auth schema.Authorization, resourceID string, action *schema.Action,
+				expectedResourceProblem resources.ResourceProblem, shouldFail bool) {
```

Comment:
We don't really need `shouldFail`. It's true if and only if `expectedResourceProblem != NoProblem`.